.github/workflows/validate-rhel-8.yml

```diff
@@ -0,0 +1,76 @@
+# Run rhel-8 unit tests in a PR triggered by a "/test [...]" comment from an organization member.
+# This avoids running untrusted and unreviewed code on self-hosted runners.
+name: unit-tests-rhel-8
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  unit-tests:
+    # restrict running of tests to trusted organization members
+    # see https://developer.github.com/v4/enum/commentauthorassociation/
+    if: startsWith(github.event.comment.body, '/test') && contains('OWNER MEMBER', github.event.comment.author_association)
+    runs-on: [self-hosted, ci-tasks, rhel-8]
+    steps:
+      - name: Get information for pull request
+        uses: octokit/request-action@v2.x
+        id: pr_info
+        with:
+          route: GET /repos/${{ github.repository }}/pulls/${{ github.event.issue.number }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      #- name: Dump PR info
+      #  run: |
+      #    echo "${{ steps.pr_info.outputs.data }}"
+
+      # the following steps only run for PRs against rhel-8 branch
+
+      # we post statuses manually as this does not run from a pull_request event
+      # https://developer.github.com/v3/repos/statuses/#create-a-status
+      - name: Create in-progress status
+        if: fromJson(steps.pr_info.outputs.data).base.ref == 'rhel-8'
```

Comment:
I'm thinking if having another job 'needing' this one would improve the state of having the ``if rhel-8`` everywhere. But maybe that's just unnecessary complication?

Reply:
I thought about it, but then indeed it would be slower at least, as that second job would need an entirely fresh VM instance. It would be more elegant, and indeed I don't particularly like the repetition. I thought we could use that approach for now, and fine-tune/optimize it later; but with this I arrived at a solution that works, and I had the impression that you'd rather land this sooner than later.

In other words, I have no strong opinion.

Reply:
Done, see below. Let me know which one you prefer.